tardis/montecarlo/packet_source.py

```diff
@@ -136,9 +136,9 @@ class BlackBodySimpleSource(BasePacketSource):
 
     Parameters
     ----------
-    radius : float64
+    radius : astropy.units.Quantity
         Initial packet radius
-    temperature : float
+    temperature : astropy.units.Quantity
```

Comment:
Making sure that all quantities are in cgs units!

Reply:
Handled by `.to_cgs()` in relevant functions